concent_api/core/payments/report.py

```diff
@@ -0,0 +1,202 @@
+from typing import List
+from typing import NamedTuple
+from typing import Tuple
+
+from django.db.models import QuerySet
+
+from common.helpers import deserialize_message
+from common.helpers import get_current_utc_timestamp
+from common.helpers import parse_datetime_to_timestamp
+from core.exceptions import SCINotSynchronized
+from core.models import Client
+from core.models import PendingResponse
+from core.models import Subtask
+from core.payments import service
+from core.payments.backends.sci_backend import TransactionType
+
+
+class PendingResponses(NamedTuple):
+    subtask_results_settled: QuerySet
+    subtask_results_rejected: QuerySet
+    force_payment_committed: QuerySet
+
+    def is_empty(self) -> bool:
```

Comment:
If you will keep the version with `len()` this and next class `empty()` method can be generalized to master class, in which in `empty()` method you iterate over attributes, but ...

Reply:
As we discussed, done :)